scripts/test/diff-exclude.cjs

```diff
@@ -55,7 +55,8 @@ const UNALIGNED_FIXTURES = [
 	"WatchSuspend.test.js",
 	"helpers/FakeDocument.js",
 	"helpers/warmup-webpack.js",
-	"hotCases/fake-update-loader.js"
+	"hotCases/fake-update-loader.js",
+	"StatsTestCases.basictest.js"
```

Comment:
These are supposed to be aligned between each other. Is there any reason behind this?